clang/include/clang/Basic/CodeGenOptions.h

```diff
@@ -208,6 +208,16 @@ class CodeGenOptions : public CodeGenOptionsBase {
               ///< larger debug info than `Basic`.
   };
 
+  enum BoolFromMem {
+    Strict,   ///< In-memory bool values are assumed to be 0 or 1, and any other
```

Comment:
This probably should be a scoped enum, else these names are in the outer name space. These names are way too generic to be in a normal enum (that is, either make this an `enum class`, or give these some sort of `ugly` prefix, VAST preference for the former).